specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -1670,6 +1671,7 @@
             "$ref": "#/definitions/ContactProfileLinkChannel"
           },
           "x-ms-identifiers": [
+            "name",
```

Comment:
Would we want to just have name in here by itself, given that the name must be unique?